lib/synapse/config_generator/haproxy.rb

```diff
@@ -1151,9 +1176,9 @@ def restart
       @next_restart += rand(@restart_jitter * @restart_interval + 1)
 
       # do the actual restart
-      res = `#{opts['reload_command']}`.chomp
+      res = `#{@opts['reload_command']}`.chomp
```

Comment:
i feel like we should be moving towards accessor methods and away from explicit instance vars, not the other way. this makes testing easier, since you can `allow(generator).to receive(:opts).and_return({})` rather than resorting to meta-programming in the specs (`generator.instance_variable_set(:ops, {})`)